mixer/adapter/cloudwatch/config/cloudwatch.yaml

```diff
@@ -10,5 +10,5 @@ spec:
CAQAAwEEAAICEgM1CBkKEAoJBAADAQQAAgIBEgM1CBQKEAoJBAADAQQAAgICEgM1FxgKDwoIBAADAQQAAgMSAzYIGQoQCgkEAAMBBAACAwESAzYIFAoQCgkEAAMBBAACAwISAzYXGAoPCggEAAMBBAACBBIDNwgSChAKCQQAAwEEAAIEARIDNwgNChAKCQQAAwEEAAIEAhIDNxARCg8KCAQAAwEEAAIFEgM4CBIKEAoJBAADAQQAAgUBEgM4CA0KEAoJBAADAQQAAgUCEgM4EBEKDwoIBAADAQQAAgYSAzkIFgoQCgkEAAMBBAACBgESAzkIEQoQCgkEAAMBBAACBgISAzkUFQoPCggEAAMBBAACBxIDOggWChAKCQQAAwEEAAIHARIDOggRChAKCQQAAwEEAAIHAhIDOhQVCg8KCAQAAwEEAAIIEgM7CBYKEAoJBAADAQQAAggBEgM7CBEKEAoJBAADAQQAAggCEgM7FBUKDwoIBAADAQQAAgkSAzwIFgoQCgkEAAMBBAACCQESAzwIEQoQCgkEAAMBBAACCQISAzwUFQoPCggEAAMBBAACChIDPQgSChAKCQQAAwEEAAIKARIDPQgMChAKCQQAAwEEAAIKAhIDPQ8RCg8KCAQAAwEEAAILEgM+CBYKEAoJBAADAQQAAgsBEgM+CBAKEAoJBAADAQQAAgsCEgM+ExUKDwoIBAADAQQAAgwSAz8IFgoQCgkEAAMBBAACDAESAz8IEAoQCgkEAAMBBAACDAISAz8TFQoPCggEAAMBBAACDRIDQAgWChAKCQQAAwEEAAINARIDQAgQChAKCQQAAwEEAAINAhIDQBMVCg8KCAQAAwEEAAIOEgNBCBYKEAoJBAADAQQAAg4BEgNBCBAKEAoJBAADAQQAAg4CEgNBExUKDwoIBAADAQQAAg8SA0IIFQoQCgkEAAMBBAACDwESA0IIDwoQCgkEAAMBBAACDwISA0ISFAoPCggEAAMBBAACEBIDQwgaChAKCQQAAwEEAAIQARIDQwgUChAKCQQAAwEEAAIQAhIDQxcZCg8KCAQAAwEEAAIREgNECB4KEAoJBAADAQQAAhEBEgNECBgKEAoJBAADAQQAAhECEgNEGx0KDwoIBAADAQQAAhISA0UIHgoQCgkEAAMBBAACEgESA0UIGAoQCgkEAAMBBAACEgISA0UbHQoPCggEAAMBBAACExIDRggeChAKCQQAAwEEAAITARIDRggYChAKCQQAAwEEAAITAhIDRhsdCg8KCAQAAwEEAAIUEgNHCB4KEAoJBAADAQQAAhQBEgNHCBgKEAoJBAADAQQAAhQCEgNHGx0KDwoIBAADAQQAAhUSA0gIGQoQCgkEAAMBBAACFQESA0gIEwoQCgkEAAMBBAACFQISA0gWGAoPCggEAAMBBAACFhIDSQgdChAKCQQAAwEEAAIWARIDSQgXChAKCQQAAwEEAAIWAhIDSRocCg8KCAQAAwEEAAIXEgNKCB0KEAoJBAADAQQAAhcBEgNKCBcKEAoJBAADAQQAAhcCEgNKGhwKDwoIBAADAQQAAhgSA0sIHQoQCgkEAAMBBAACGAESA0sIFwoQCgkEAAMBBAACGAISA0saHAoPCggEAAMBBAACGRIDTAgdChAKCQQAAwEEAAIZARIDTAgXChAKCQQAAwEEAAIZAhIDTBocCg8KCAQAAwEEAAIaEgNNCBoKEAoJBAADAQQAAhoBEgNNCBQKEAoJBAADAQQAAhoCEgNNFxkKuAEKBgQAAwECABIDUQYUGqgBIFRoZSB1bml0IG9mIHRoZSBtZXRyaWMuIE11c3QgYmUgdmFsaWQgY2xvdWR3YXRjaCB1bml0IHZhbHVlLgogW0Nsb3VkV2F0Y2ggZG9jc10oaHR0cHM6Ly9kb2NzLmF3cy5hbWF6b24uY29tL0FtYXpvbkNsb3VkV2F0Y2gvbGF0ZXN0L0FQSVJlZmVyZW5jZS9BUElfTWV0cmljRGF0dW0uaHRtbCkKCg8KBwQAAwECAAQSBFEGTgcKDgoHBAADAQIABhIDUQYKCg4KBwQAAwECAAESA1ELDwoOCgcEAAMBAgADEgNREhMKOwoEBAACAhIDVQQeGi4gVGhlIG5hbWUgb2YgdGhlIGxvZyBncm91cCBpbiBjbG91ZHdhdGNobG9ncy4KCg0KBQQAAgIEEgRVBFIFCgwKBQQAAgIFEgNVBAoKDAoFBAACAgESA1ULGQoMCgUEAAICAxIDVRwdCjwKBAQAAgMSA1gEHxovIFRoZSBuYW1lIG9mIHRoZSBsb2cgc3RyZWFtIGluIGNsb3Vkd2F0Y2hsb2dzLgoKDQoFBAACAwQSBFgEVR4KDAoFBAACAwUSA1gECgoMCgUEAAIDARIDWAsaCgwKBQQAAgMDEgNYHR4KSAoEBAACBBIDWwQiGjsgQSBtYXAgb2YgSXN0aW8gbG9nZW50cnkgbmFtZSB0byBDbG91ZFdhdGNoIGxvZ2VudHJ5IGluZm8uCgoNCgUEAAIEBBIEWwRYHwoMCgUEAAIEBhIDWwQYCgwKBQQAAgQBEgNbGR0KDAoFBAACBAMSA1sgIQoMCgQEAAMDEgRdBWEFCgwKBQQAAwMBEgNdDRQKzAEKBgQAAwMCABIDYAYiGrwBIEEgZ29sYW5nIHRleHQvdGVtcGxhdGUgdGVtcGxhdGUgdGhhdCB3aWxsIGJlIGV4ZWN1dGVkIHRvIGNvbnN0cnVjdCB0aGUgcGF5bG9hZCBmb3IgdGhpcyBsb2cgZW50cnkuCiBJdCB3aWxsIGJlIGdpdmVuIHRoZSBmdWxsIHNldCBvZiB2YXJpYWJsZXMgZm9yIHRoZSBsb2cgdG8gdXNlIHRvIGNvbnN0cnVjdCBpdHMgcmVzdWx0LgoKDwoHBAADAwIABBIEYAZdFQoOCgcEAAMDAgAFEgNgBgwKDgoHBAADAwIAARIDYA0dCg4KBwQAAwMCAAMSA2AgIWIGcHJvdG8z
```

Comment:
I am curious what this is and how to decode it.

Reply:
This is generated content representing the shape of the API. I think it's a serialized proto descriptor.

Reply:
When I review a PR containing base64 I like to decode it and make sure there are no passwords.  If I do `kubectl -n istio-system get adapter cloudwatch -o jsonpath='{.spec.config}' | base64 --decode | protoc --decode_raw` I get something.    I looked around and couldn't figure out what to pass to get a type specific decode.